_pages/activities/skiing.md

```diff
@@ -10,7 +10,105 @@
 
 MITOC has an active group of cross country and backcountry skiers, with trips typically running from January through May. If you are interested in learning to cross country and/or backcountry ski, we run introductory trips during [Winter School](/events/winter-school). In addition, MITOCers organise trips not only here in the Northeast but across the US and beyond. Join the [skiing mailing list](http://mailman.mit.edu/mailman/listinfo/mitoc-bcski) and become a member of our skiing community. Upcoming trips and events will be announced on this list.
 
-You can rent cross-country, backcountry, telemark, and AT ski equipment from MITOC. Please see the [gear rental information page](/rentals) for details. MITOC does not have downhill/alpine ski equipment for rental.
+### Personal trips
+
+You can rent cross-country, backcountry, telemark, and AT ski equipment from MITOC. We encourage personal trips and rent out our fleet for personal use, except during [Winter School](/events/winter-school). Please use our gear responsibly and adhere to the club rules. **MITOC does not have downhill/alpine ski equipment for rental.**
+
+<p align="center">
+    <img style="max-width: 50%; height: auto;" alt="Our skis are for backcountry use ONLY. Resort skiing is not allowed unless the user is uphill skiing. Do not aggressively load the equipment. MITOC has a strict no avalanche terrain policy. There can be no exceptions." src="https://github.com/user-attachments/assets/518dfb6c-a22b-4fe2-8b88-08b396478fd2" />
```

Comment:
I'd missed this (CC @mprat and @tsverk ) but we don't really want to host images on `github.com` on MITOC's web site -- you'll want to put these images into the static site itself.